docs/site/content/docs/latest/upgrade-kapp-controller.md

```diff
@@ -0,0 +1,182 @@
+# Upgrade kapp-controller
+
+Tanzu Community Edition provides a built in version of kapp-controller that is dictated
+by the Bill of Materials (BOM) of the Tanzu Kubernetes Release (TKR) in use.
+
+It is possible to upgrade kapp-controller version provided. Depending on your type of
+cluster you will need to follow one of the available approaches.
```

Comment:
might be clearer to say:
Depending on whether you are running a standalone or management cluster, you will need to follow one of the following approaches.